api.go

```diff
@@ -506,6 +686,10 @@ func (r *Raft) RemovePeer(peer ServerAddress) Future {
 // If nonzero, timeout is how long this server should wait before the
 // configuration change log entry is appended.
 func (r *Raft) AddVoter(id ServerID, address ServerAddress, prevIndex uint64, timeout time.Duration) IndexFuture {
+	if r.protocolVersion < 1 {
+		return errorFuture{ErrUnsupportedProtocol}
+	}
+
```

Comment:
I thought that while this server is in what used to be called version 1, it must never generate a LogConfiguration with ID != addr. Is that correct?